### DISCUSSION TITLE
Dark mode dropdown rendering problem

### DISCUSSION THREAD

**Discussion by jedbrown** [pradyunsg/furo#548] [Potential Issue] [answered]
[Our dropdowns](https://ratel.micromorph.org/doc/modeling/methods/hyperelasticity/#equation-eq-diff-p) ([source](https://gitlab.com/micromorph/ratel/-/blob/main/doc/modeling/methods/hyperelasticity.md?plain=1#L71)) are all white (thus unreadable) in dark-mode Furo.

![image](https://user-images.githubusercontent.com/3303/194798767-c003456f-897c-4492-a4f0-378110a3a0cc.png)

The dropdowns render properly in light mode.

![image](https://user-images.githubusercontent.com/3303/194798823-10c20bf3-f382-4cc2-bba6-7af50b170c89.png)

**Comment by pradyunsg** [comment, marked as answer]
This is due to the use of `sphinx-panels`. The sphinx-panels repository contains the following warning in the README:

> 🚨This repository is not actively maintained. Use [sphinx-design](https://github.com/executablebooks/sphinx-design) instead! See [the migration guide](https://sphinx-design.readthedocs.io/en/latest/get_started.html#migrating-from-sphinx-panels) and https://github.com/executablebooks/sphinx-design/issues/51 for more information.🚨

Migrating to `sphinx-design` will resolve this.

**Reply by pradyunsg** [reply]
https://github.com/executablebooks/sphinx-panels

**Reply by jedbrown** [reply]
Wonderful, thank you!
